11.2_Основы_работы_со_списками/11.2.9.py

```diff
@@ -0,0 +1,5 @@
+numbers1 = [1, 2, 3]
+numbers2 = [6]
+numbers3 = [7, 8, 9, 10, 11, 12, 13]
+
+print(numbers1 + numbers1 + numbers2 + numbers2 + numbers2 + numbers2 + numbers2 + numbers2 + numbers2 + numbers2 + numbers2 + numbers3)
```

Comment:
Невнимательно читаем теорию :)

Списки, как и строки, можно складывать и умножать:
```suggestion
print(numbers1 * 2 + numbers2 * 9 + numbers3)
```